### PR TITLE
Allow unauthorized bodies access

### PR DESCRIPTION
**Pull request by serge1peshcoff**
Fixes MEMB-680

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -24,7 +24,7 @@ config :tesla, adapter: Tesla.Mock
 config :omscore, Omscore.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: "postgres",
-  password: "postgres",
+  password: "5ecr3t",
```
Comment (linuxbandit) [inline config/test.exs]: i think this should be put back to postgres to not break what's in prod

Comment (serge1peshcoff) [reply config/test.exs]: it's testing, it shouldn't. I didn't touch any other files and this is just to make it consistent.

Comment (serge1peshcoff) [reply config/test.exs]: I can delete it, but for prod it will make no difference as the only place testing env is used is CI

Comment (linuxbandit) [reply config/test.exs]: true

**Pull request merged**